### PR TITLE
Chore: Add resin component to sidebar

### PR DESCRIPTION
**Pull request by JustinHoldstock**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @JustinHoldstock has signed the CLA. Thanks for the pull request!

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #471

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

File: src/components/ContentSidebar/ContentSidebar.js [hunk 1]
```diff
@@ -93,6 +93,9 @@ const activityFeedInlineError: Errors = {
         content: messages.activityFeedItemApiError
     }
 };
+
+const RESIN_INFOPANE_TAG = 'infopane';
```
Comment (priyajeet) [inline src/components/ContentSidebar/ContentSidebar.js]: src/interactionTargets.js

Comment (priyajeet) [reply src/components/ContentSidebar/ContentSidebar.js]: also call it 'sidebar' to be consistent

Comment (priyajeet) [inline src/components/ContentSidebar/ContentSidebar.js]: Wait this is adding a component. We aren't adding any resin tags other than target in UI elements.

Comment (jeremypress) [reply src/components/ContentSidebar/ContentSidebar.js]: Is there a way we could allow for generic attributes in content preview that get passed in by EUA? Similar to how many BRUI components do?

Comment (jeremypress) [reply src/components/ContentSidebar/ContentSidebar.js]: that won't help much since we already have the page labeled...

Comment (jeremypress) [reply src/components/ContentSidebar/ContentSidebar.js]: We could pass it in the sidebar props and apply ...rest to the DOM element like BRUI does in many instances

Comment (JustinHoldstock) [reply src/components/ContentSidebar/ContentSidebar.js]: `infopane` was what is requested as the component name, to be consistent with Webapp. Leaving as is

**Pull request closed without merging**